climatedata_api/map.py

```diff
@@ -241,3 +243,59 @@ def get_id_list_from_points(compressed_points):
             break
 
     return gids.tolist()
+
+def get_s2d_release_date(var, freq):
+    """
+    Return the release date of the forecast data associated to a given variable and frequency
+    curl 'http://localhost:5000/get-s2d-release-date/air_temp/seasonal'
+    """
+    try:
+        if var not in app.config['S2D_VARIABLES']:
+            raise ValueError
+        if freq not in app.config['S2D_FREQUENCIES']:
+            raise ValueError
+    except ValueError:
+        return "Bad request", 400
```

Comment:
J'irais peut-être plus avec un simple "if"


```suggestion
    if (var not in app.config['S2D_VARIABLES']) or (freq not in app.config['S2D_FREQUENCIES'])
        return "Bad request", 400
```